pkg/agent/apiserver/handlers/podinterface/handler_test.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package podinterface
+
+import (
+	"encoding/json"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+
+	interfacestore "github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	mock_interfacestore "github.com/vmware-tanzu/antrea/pkg/agent/interfacestore/testing"
```

Comment:
no underscore in package and its alias.
You can use `interfacestoretest` like others.

Reply:
Updated